### ISSUE TITLE
fading_edge_scrollview update

### ISSUE THREAD

**Issue by hatim-essadeq** [MarcelGarus/marquee#98]
in Flutter 3.22.0 : we have this error : The argument type 'PageController?' can't be assigned to the parameter type 'ScrollController' because 'PageController?' is nullable and 'ScrollController' isn't.
you should update the package of fading_edge_scrollview to resolve the problem.

thanks in advance.

**Comment by ipasaris** [comment]
I also encountered the same issue. It's a bummer. Had to downgrade back to Flutter v3.19.6. I'm missing out on a lot of cool features. Please make the necessary code changes.

Thanks in advance.

**Comment by Luism96xd** [comment]
I have the same issue, I also using Flutter 3.22.0

**Comment by ShokhrukhbekYuldoshev** [comment]
Temporary fix

```yaml
# Fix for marquee dependency after Flutter 3.2.2 upgrade
dependency_overrides:
    fading_edge_scrollview: ^4.1.1
```

**Comment by socheatech** [comment]
I have the same issue, I also using Flutter 3.22.0

**Comment by 6Yolo6** [comment]
same error！

**Comment by hakama99** [comment]
Received notification from Android that the app must be updated to SDK 34 within fifty days, necessitating the upgrade of Flutter to version 3.22.2.

`# Fix for marquee dependency after Flutter 3.2.2 upgrade
dependency_overrides:
    fading_edge_scrollview: ^4.1.1`

This works for me.

**Comment by realitymolder** [comment]
have been updated on the new version 2.3 :)
